packages/mask/src/social-network-adaptor/facebook.com/injection/ToolbarUI.tsx

```diff
@@ -4,6 +4,8 @@ import { useMemo } from 'react'
 
 const fbBreakPoint = 700 /** px */
 
+const FB_BREAKPOINT = 1159
```

Comment:
This number seems quite big for a small breakpoint. Please test on Facebook or search their docs for a certain number.

Reply:
<img width="900" alt="image" src="https://github.com/DimensionDev/Maskbook/assets/72531217/2b06dc9f-e42f-4e94-81cd-3c059fe14297">